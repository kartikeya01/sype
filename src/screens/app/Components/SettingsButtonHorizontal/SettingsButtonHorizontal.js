import {IconButton} from 'react-native-paper';
import React from 'react';

export default class SettingsButtonHorizontal extends React.Component {
  state = {
    iconSize: 30,
    settingsButtonHorizontalIcon: 'dots-horizontal',
  };

  handleSettingsClick = () => {};

  render() {
    return (
      <IconButton
        icon={this.state.settingsButtonHorizontalIcon}
        size={this.state.iconSize}
        onPress={this.handleSettingsClick}
      />
    );
  }
}
