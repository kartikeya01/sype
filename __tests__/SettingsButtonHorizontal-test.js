import 'react-native';
import React from 'react';
import SettingsButtonHorizontal from '@screens/app/Components/SettingsButtonHorizontal/SettingsButtonHorizontal';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<SettingsButtonHorizontal />).toJSON();
  expect(tree).toMatchSnapshot();
});
