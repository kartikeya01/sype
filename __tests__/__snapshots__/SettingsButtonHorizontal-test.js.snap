// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityRole="button"
  accessibilityStates={Array []}
  accessible={true}
  focusable={false}
  hitSlop={
    Object {
      "bottom": 6,
      "left": 6,
      "right": 6,
      "top": 6,
    }
  }
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "overflow": "hidden",
      },
      Array [
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "margin": 6,
          "overflow": "hidden",
        },
        Object {
          "borderRadius": 22.5,
          "height": 45,
          "width": 45,
        },
        undefined,
        undefined,
      ],
    ]
  }
>
  <View
    style={null}
  >
    <Text
      accessibilityElementsHidden={true}
      allowFontScaling={false}
      importantForAccessibility="no-hide-descendants"
      pointerEvents="none"
      style={
        Array [
          Object {
            "color": "#000000",
            "fontSize": 30,
          },
          Array [
            Object {
              "transform": Array [
                Object {
                  "scaleX": 1,
                },
              ],
            },
            Object {
              "backgroundColor": "transparent",
            },
          ],
          Object {
            "fontFamily": "Material Design Icons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      ï‡˜
    </Text>
  </View>
</View>
`;
